.github/workflows/build.yml

```diff
@@ -0,0 +1,80 @@
+name: Build Binaries
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        type: string
+      addon-name:
+        type: string
+        default: hex_map
+    outputs:
+      artifact-name:
+        value: ${{ inputs.addon-name }}
```

Comment:
Is there a reason we have to copy this value from input to output?  Is input not accessible via `needs` similar to `needs.build.output`?